spec/services/create_requested_account_spec.rb

```diff
@@ -34,20 +34,23 @@
     stub_account_creation(wiki: home_wiki)
     allow(UserImporter).to receive(:new_from_username).and_return(user)
     expect(subject.result[:success]).not_to be_nil
+    expect(subject.result[:result_description]).not_to be_nil
```

Comment:
This could be better. Instead of just *not nil*, it should look for some specific text that will give us a clear indication that it worked as expected, such as looking for the the string "Success:" or "Failure:" within the description.

Reply:
Done!